src/DisplayBoard.cpp

```diff
@@ -101,20 +115,72 @@ std::string drawColumns(const unsigned int maxElementsInRows,
     for (int i = heightCol; i > 0; --i)
     {
         columns += PADDING + drawPadding(maxElementsInRows);
-        columns += drawEmptyLine(sizeCols);
+        columns += VERTICAL;
+        columns += drawRowOfColumns(i, stringClues);
+        columns += VERTICAL;
         columns += "\n";
     }
+    return columns;
+}
+
+std::vector<std::string> getFormattedColumns(const unsigned int sizeCols,
+                                              const Lines & cluesCols)
+{
+    std::vector<std::string> stringLines;
+    if(not cluesCols.empty())
+        for(const auto & clueCol : cluesCols)
+        {
+            std::string strline {};
+            std::transform(clueCol.begin(), clueCol.end(), std::back_inserter(strline),
+                         [&](const auto i)
+            {
+                return std::to_string(i).at(0);
+            });
+            if(strline.size() < sizeCols)
+                while (strline.size() != sizeCols) {
+                    strline.push_back(PADDING);
+                }
+            stringLines.emplace_back(strline);
+        }
+    return stringLines;
+}
 
+std::string drawRowOfColumns(const unsigned pos,
+                       const std::vector<std::string> & clues)
+{
+    std::string columns;
+    std::string s = " ";
+    if(not clues.empty())
+        for(const auto & line : clues)
+            columns += s + line.at(pos - 1);
```

Comment:
maybe just use PADDING instead of variable s? s is temporary const in this loop.
columns += PADDING + line.at(pos - 1);